flashinfer_trace/tests/definitions/test_moe_fp8.py

```
@@ -0,0 +1,204 @@
+def generate_moe_fp8_inputs(
+    seq_len: int,
+    num_experts_global: int = NUM_EXPERTS_GLOBAL,
+    num_local_experts: int = NUM_EXPERTS_LOCAL,
+    hidden_size: int = HIDDEN_SIZE,
+    intermediate_size: int = INTERMEDIATE_SIZE,
+    local_expert_offset: int = 0,
+    routed_scaling_factor: float = 2.5,
+    device: str = "cuda",
+):
+    """Generate random inputs for MoE FP8 testing."""
+    T, H, I = seq_len, hidden_size, intermediate_size
```

Ruff (0.14.13)

Ambiguous variable name: `I`

(E741)